tests/automation_framework/src/libs/avalon_libs.py

```diff
@@ -27,6 +27,23 @@
 class AvalonImpl():
 
     def worker_lookup(self):
+        """
+        Submit's WorkerLookup request to submit_request_listener
```

Comment:
Have this line same as worker retrieve: Submit WorkerRetrieve request to listener/sdk